### ISSUE TITLE
Binary builds - releases 1.4.0b1 & 2.0.0b8: possible to (re)add "borg-linux-glibc224"? (vagrant stretch)

### ISSUE THREAD

**Issue by trendymail** [borgbackup/borg#8132]
Hello!

## Have you checked borgbackup docs, FAQ, and open GitHub issues?

A little: 00_README.txt provided with standalone binaries, https://github.com/borgbackup/borg/issues/7459, https://github.com/borgbackup/borg/pull/8038 and https://github.com/borgbackup/borg/issues/7989.

## Is this a BUG / ISSUE report or a QUESTION?

A polite and helpless (?) request to "revive" borg-linuxold64 for 1.4.x & 2.0.x branches (if possible) 

## System information.

OpenVZ 7: ldd --version 2.17 (cannot upgrade libc)
OpenVZ 8: dead project, won't be released (alpha)
OpenVZ 9: not ready at all (beta, numerous bugs)

#### Your borg version (borg -V).

1.2.7:  borg-linuxold64 (no issues, working fine)
1.4.0:  borg-linux-glibc224 (missing/never added)
2.0.0:  borg-linuxold64 (removed in beta 7)

## Long story short

Stretch Vagrant box was fixed for branch 1.2.X but, due to numerous changes in 1.4.x and 2.0.X, I honestly do not even know if this is possible to build borg from source with such an old libc (2.17).

Please, feel free to close this request as "wontfix" if I missed something obvious.

There are many open or closed issues about Stretch (libc, openssl, vagrant, etc) but I did not read them all (sorry).

Have a great day! :)

**Comment by ThomasWaldmann** [comment]
You can read the git changelog, there were reasons why I removed that.

It was already a pain and hard to fix in 1.2, but not gonna continue that in 1.4 / master.

I plan to provide security and severe bug fixes for 1.2 for a while after 1.4 release, so you can just stay on 1.2.x.

**Comment by ThomasWaldmann** [comment]
BTW, the main point of 1.4 was to provide a "refresh" until borg2 is ready: getting rid of some legacy, getting rid of bundled stuff, add exit codes. Other than that, it is quite similar to 1.2.7.

**Comment by trendymail** [comment]
Understood!

I will stick to 1.2.x version for OpenVZ 7 (it works very well) and use 1.4.x/2.0.x versions within containers (Debian 12 Bookworm).

Many thanks for your answers. :)

I should have read the changelog,..
